src/category_theory/limits/limits.lean

```diff
@@ -963,6 +977,19 @@ end
 
 section lim_functor
 
+/-- Functoriality of limits, stated for `is_limit`. -/
+def is_limit_map {F G : J ⥤ C} (c : cone F) {d : cone G} (hd : is_limit d) (α : F ⟶ G) : c.X ⟶ d.X :=
+hd.lift
+  { X := c.X,
+    π :=
+    { app := λ j, c.π.app j ≫ α.app j,
+      naturality' := λ j j' f,
+        by erw [id_comp, assoc, ←α.naturality, ←assoc, c.w] } }
```

Comment:
I could be mistaken, but isn't this new cone you construct just a particular whiskering? If so, it's probably worth using that definition instead of re-doing it.

Reply:
Yes, thanks!